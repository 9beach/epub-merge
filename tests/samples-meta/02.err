[90mìž‘ê°€: ë°°ìˆ˜ì•„ -> ë°°ìˆ˜ì•„ [ë°°, ìˆ˜ì•„]; Suah Bae [Bae, Suah][0m
[90më²ˆì—­: <empty> -> Deborah Smith[0m
[90mì œëª©: ë±€ê³¼ ë¬¼ -> ë±€ê³¼ë¬¼ (Snake And Water)[0m
