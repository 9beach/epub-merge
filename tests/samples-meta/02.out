제목: 뱀과물 (Snake And Water)
작가: 배수아 [배, 수아]
작가: Suah Bae [Bae, Suah]
번역: Deborah Smith
출판일: 2017-12-07
언어: ko
